common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -387,7 +387,7 @@ def get_html(self):
         return self.system.render_template('video.html', context)
 
 
-@XBlock.wants("request_cache", "settings", "completion")
+@XBlock.wants("settings", "completion")
```

Comment:
Does this mean the request cache has been removed as an XBlock runtime service? Won't that potentially break any external XBlocks that use it?

Reply:
No, I haven't removed it as a runtime service.  Rather, I just changed the video_module to use the common `request_cached` decorator.  That decorator, by default, uses the edx-django-util's `RequestCache`.

Note: I made this change before realizing that I needed to introduce a `request_cache_getter` for the modulestore's `has_changes` method.  Now that we have that, I can revert back to using the request_cache from the XBlock runtime service.  However, is that really needed at this point - given that we're Ok with XBlock's using Django/services directly?

Reply:
Ok. I updated this code to use the new `request_cache_getter` parameter, allowing me to revert the changes for getting the request_cache from the runtime service.